tests/gatewayapitests/ingestion/httproute_test.go

```diff
@@ -249,3 +250,43 @@ func TestHTTPRouteGatewayWithEmptyHostname(t *testing.T) {
 	akogatewayapitests.TeardownGatewayClass(t, gatewayClassName)
 	waitAndverify(t, gwClassKey)
 }
+
+func TestHTTPRouteGatewayWithRegexPath(t *testing.T) {
+	gatewayClassName := "gateway-class-07"
+	gatewayName := "gateway-07"
+	httpRouteName := "httproute-07"
+	gwKey := "Gateway/" + DEFAULT_NAMESPACE + "/" + gatewayName
+	gwClassKey := "GatewayClass/" + gatewayClassName
+	namespace := "default"
+	ports := []int32{8080}
+	key := "HTTPRoute" + "/" + namespace + "/" + httpRouteName
+
+	// gatewayclass
+	akogatewayapiobjects.GatewayApiLister().UpdateGatewayClass(gatewayClassName, true)
+	akogatewayapitests.SetupGatewayClass(t, gatewayClassName, akogatewayapilib.GatewayController)
+	t.Logf("Created GatewayClass %s", gatewayClassName)
+	waitAndverify(t, gwClassKey)
+
+	// Gateway without hostname
+	listeners := akogatewayapitests.GetListenersV1(ports, true, false)
```

Comment:
Is there a reason why we are testing it for gateway with empty hostname ?

Reply:
There is no specific reason for this, any Gateway should work. So added empty hostname Gateway